app/build.gradle

```diff
@@ -73,7 +73,7 @@ dependencies {
     compile 'org.greenrobot:eventbus:3.0.0'
 
     testCompile 'junit:junit:4.12'
-    testCompile 'org.mockito:mockito-core:2.7.22'
+    androidTestCompile 'org.mockito:mockito-core:2.7.22'
```

Comment:
@jverbraeken You wanne have **both** `testCompile 'org.mockito:mockito-core:2.7.22'` and `androidTestCompile 'org.mockito:mockito-core:2.7.22'`...